packages/marko-web-theme-monorail/scss/components/_identity-x.scss

```diff
@@ -34,4 +34,15 @@
       border-radius: 4px;
     }
   }
+  &--profile {
+    .success-message {
+      color: pink;
```

Comment:
Why pink? just use `text-success` class and allow the theme to dictate what that color actually is.